### PR TITLE
Update motoko-introduction.adoc

### PR DESCRIPTION
**Pull request by christyjacob4**
Fixed some typos in the docs

### CODE REVIEW

File: doc/modules/language-guide/pages/motoko-introduction.adoc [hunk 1]
```diff
@@ -14,7 +14,7 @@ Although aimed squarely at the {IC}, its design is general enough to support fut
 == Asynchronous messaging and type sound execution
 
 {proglang} permits modern programming idioms, including special programming abstractions for distributed applications (dapps).
-Each dapp consists of one or more  _actor_s that communicate solely by asynchronous message passing. The state of an actor is isolated from all other actors, supporting distribution. There is no way to share state between several actors. 
+Each dapp consists of one or more  _actors_ that communicate solely by asynchronous message passing. The state of an actor is isolated from all other actors, supporting distribution. There is no way to share state between several actors.
```
Comment (ggreif) [inline doc/modules/language-guide/pages/motoko-introduction.adoc:17]: @crusso I think the intention here was to set `actor` as teletype and the "s" (for plural) as regular. @christyjacob4 can you drop a small .PNG how this revered before and after?

Comment (christyjacob4) [reply doc/modules/language-guide/pages/motoko-introduction.adoc:17]: Absolutely 👍

File: doc/modules/language-guide/pages/motoko-introduction.adoc [hunk 2]
```diff
@@ -24,14 +24,14 @@ Consequently, entire classes of common programming pitfalls that are common in o
...
-== Each canister smart constract as an _actor_
+== Each canister smart contract as an _actor_
 
 {proglang} provides an *actor-based* programming model to developers to express _services_, including those of canister smart contracts on the link:../developers-guide/concepts/what-is-ic{outfilesuffix}#ic-overview[{IC}].
 
 An actor is similar to an object, but is special in that its state is completely isolated, and all its interactions with the world are by _asynchronous_ messaging.
 
 All communication with and between actors involves passing messages asynchronously over the network using the Internet Computer's messaging protocol.
-An actor’s messages are processed in sequence, so state modifications never admit race conditions (unless explicitly allowed by punctuating `await`s).
+An actor’s messages are processed in sequence, so state modifications never admit race conditions (unless explicitly allowed by punctuating `await` s).
```
Comment (ggreif) [inline doc/modules/language-guide/pages/motoko-introduction.adoc:34]: Make `actor` teletype here? Also, are backticks allowed in `.adoc`?

File: doc/modules/language-guide/pages/motoko-introduction.adoc [hunk 3]
```diff
@@ -95,7 +95,7 @@ We do not show an example of this "`notify`" pattern above, since it uses callba
 To address the need for clarity and simplicity, {proglang} adopts the increasingly-common program constructs `async` and `await`, which afford the programmer a _structured_ language for describing potentially-complex asynchronous dependency graphs.
 
 The link:language-manual{outfilesuffix}#exp-async[async] syntax introduces futures. A future value
-represents a _"`promise`" of a result that will be delivered, asynchronously, sometime in the future_ (not shown in the first example above).
+represents a _" `promise` " of a result that will be delivered, asynchronously, sometime in the future_ (not shown in the first example above).
```
Comment (ggreif) [inline doc/modules/language-guide/pages/motoko-introduction.adoc:98]: No idea how thus is supposed to come out... @crusso I'll leave this to you :-)

File: doc/modules/language-guide/pages/motoko-introduction.adoc (hunk 1 above)
Comment (christyjacob4) [reply doc/modules/language-guide/pages/motoko-introduction.adoc:17]: Before ( in page https://smartcontracts.org/docs/language-guide/motoko-introduction.html#_asynchronous_messaging_and_type_sound_execution )

Screenshot 2021-12-07 at 3 57 04 AM
 

After ( in page https://github.com/christyjacob4/motoko/blob/patch-2/doc/modules/language-guide/pages/motoko-introduction.adoc#asynchronous-messaging-and-type-sound-execution )
Screenshot 2021-12-07 at 3 59 30 AM


Unfortunately I could not find the contribution guidelines to generate the docs locally. Would it make sense for me to add a `Contributing.md` to this repo ? If yes, where can we have a discussion on what topics need to be covered in the guidelines (apart from the obvious how to setup / build / run docs locally ) ?

File: doc/modules/language-guide/pages/motoko-introduction.adoc (hunk 3 above)
Comment (christyjacob4) [reply doc/modules/language-guide/pages/motoko-introduction.adoc:98]: When I looked at the markdown syntax , the below rendered output made most sense to me. 
Screenshot 2021-12-07 at 4 01 51 AM

 So I've edited this doc to render a code quote as per GitHub Markdown rules. 
Screenshot 2021-12-07 at 4 04 43 AM

Refer : https://github.com/christyjacob4/motoko/blob/patch-2/doc/modules/language-guide/pages/motoko-introduction.adoc#syntactic-forms-async-and-await

File: doc/modules/language-guide/pages/motoko-introduction.adoc (hunk 2 above)
Comment (christyjacob4) [reply doc/modules/language-guide/pages/motoko-introduction.adoc:34]: Im a bit confused about the usage of teletype, italics, bold and so on. 
I read the language guides ( https://smartcontracts.org/docs/language-guide/about-this-guide.html#_documentation_conventions ) but it seems the usage bold, italics, and so on, don't always comply with the guidelines and are being used in a way the author interpreted it the best.

File: doc/modules/language-guide/pages/motoko-introduction.adoc (hunk 1 above)
Comment (ggreif) [reply doc/modules/language-guide/pages/motoko-introduction.adoc:17]: I think you are doing the right thing. Just file a pull request.

Comment (ggreif) [reply doc/modules/language-guide/pages/motoko-introduction.adoc:17]: I found a reference that could help: https://docs.asciidoctor.org/asciidoc/latest/text/quotation-marks-and-apostrophes/#possessive-monospace
Possibly one has towrite double backticks around `actor` like this:
``` adoc
  ... ``actor``s ...
```

Comment (ggreif) [reply doc/modules/language-guide/pages/motoko-introduction.adoc:17]: If you want to see how this is rendered, you can select "View file" from the menu right to the "Viewed" checkbox above.

Comment (christyjacob4) [reply doc/modules/language-guide/pages/motoko-introduction.adoc:17]: going by the underscore character that was already present before my chance. I assume the intention was to have actors in italics _actors_ . 

Correct me if I'm wrong.

File: doc/modules/language-guide/pages/motoko-introduction.adoc (hunk 2 above)
Comment (ggreif) [reply doc/modules/language-guide/pages/motoko-introduction.adoc:34]: Single backticks seem to be _emphasis_.

File: doc/modules/language-guide/pages/motoko-introduction.adoc (hunk 1 above)
Comment (crusso) [reply doc/modules/language-guide/pages/motoko-introduction.adoc:17]: No, that right. Sorry to be so late to the party.

File: doc/modules/language-guide/pages/motoko-introduction.adoc (hunk 2 above)
Comment (crusso) [reply doc/modules/language-guide/pages/motoko-introduction.adoc:34]:
```suggestion
An actor's messages are processed in sequence, so state modifications never admit race conditions (unless explicitly allowed by punctuating ``await``s).
```

File: doc/modules/language-guide/pages/motoko-introduction.adoc (hunk 3 above)
Comment (crusso) [reply doc/modules/language-guide/pages/motoko-introduction.adoc:98]:
```suggestion
represents a _promise_ of a result _that will be delivered, asynchronously, sometime in the future_ (not shown in the first example above).
```

File: doc/modules/language-guide/pages/motoko-introduction.adoc (hunk 1 above)
Comment (crusso) [reply doc/modules/language-guide/pages/motoko-introduction.adoc:17]:
```suggestion
Each dapp consists of one or more  _actors_ that communicate solely by _asynchronous message passing_. The state of an actor is isolated from all other actors, supporting distribution. There is no way to share state between several actors. 
```

File: doc/modules/language-guide/pages/motoko-introduction.adoc [hunk 4]
```diff
@@ -24,14 +24,14 @@ Consequently, entire classes of common programming pitfalls that are common in o
...
-== Each canister smart constract as an _actor_
+== Each canister smart contract as an _actor_
 
 {proglang} provides an *actor-based* programming model to developers to express _services_, including those of canister smart contracts on the link:../developers-guide/concepts/what-is-ic{outfilesuffix}#ic-overview[{IC}].
 
 An actor is similar to an object, but is special in that its state is completely isolated, and all its interactions with the world are by _asynchronous_ messaging.
 
 All communication with and between actors involves passing messages asynchronously over the network using the Internet Computer's messaging protocol.
-An actor’s messages are processed in sequence, so state modifications never admit race conditions (unless explicitly allowed by punctuating `await`s).
+An actor's messages are processed in sequence, so state modifications never admit race conditions (unless explicitly allowed by punctuating ``await``s).
```
Comment (crusso) [inline doc/modules/language-guide/pages/motoko-introduction.adoc:34]:
```suggestion
An actor's messages are processed in sequence, so state modifications never admit race conditions (unless explicitly allowed by punctuating `await` expressions).
```

**Comment by crusso** [comment]
Thanks for the suggestions, and sorry for responding so late.

**Review by crusso — APPROVED** [review]
LGTM

**Comment by nomeata** [comment]
Claudio, external contributers won't be able to merge or tag, so we have to do that.

**Comment by christyjacob4** [comment]
Synced with changes from master

**Pull request merged**